recipes/sqlalchemy-risingwave/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sqlalchemy-risingwave" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sqlalchemy-risingwave-{{ version }}.tar.gz
+  sha256: 856a3c44b98cba34d399c3cc9785a74896caca152b3685d87553e4210e3e07a4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - sqlalchemy >=1.4,<2
+
+test:
+  imports:
+    - sqlalchemy_risingwave
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/risingwavelabs/risingwave
+  summary: RisingWave dialect for SQLAlchemy
+  dev_url: https://github.com/risingwavelabs/sqlalchemy-risingwave
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - gforsyth
+    - KeXiangWang
```

Comment:
```suggestion
    - KeXiangWang
    - yuhao-su
```

Reply:
Also add my colleagues please if possible.

Reply:
I can add them once they comment in the issue, otherwise we can add them in a followup

Reply:
yeah, let's add him later.